packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -70,8 +77,9 @@ class Layout extends React.Component {
           <main id="main-container" className={cx('main-container')}>
             <ResponsiveElement defaultElement={overlay} responsiveTo="window" medium={<div />} />
             <div className={cx('main-container-inner')}>
-              <div className={cx('nav-burger')}>
+              <div className={cx('nav-burgerbar')}>
                 <Nav.Burger handleClick={this.toggleNav} />
+                {mobileLogo && <div className={cx('mobile-logo')}>{mobileLogo}</div>}
```

Comment:
why not just apply the logo with CSS?

Reply:
`why not just apply the logo with CSS?`
I'd like to create a logo holder here to control the position of logo instead, because we know we'd like to keep the logo center justified but we don't have a reference to the logo in CSS at this point.

`Does it need to be clickable?`
Not required. Users can passed in an clickable logo if they want. The logo holder here is for positioning.

Reply:
>  we know we'd like to keep the logo center justified but we don't have a reference to the logo in CSS at this point.

It would be a CSS custom property.. so technically you would.